src/main/haskell/language-kore/src/Data/Kore/ASTPrettyPrint.hs

```diff
@@ -624,6 +624,16 @@ instance
                 "sentenceSortAttributes" (sentenceSortAttributes sa)
```

Comment:
Pretty print test? Either as a unittest or as a .kore file in the test/ directory.